step.yml

```diff
@@ -1,6 +1,38 @@
 title: "[BETA] Xcode Build for testing for iOS"
 summary: Performs xcodebuild's build-for-testing action
-description: Performs xcodebuild's build-for-testing action, to build tests without running them.
+description: |- 
+  The Step runs Xcode's `xcodebuild` command with the `build-for-testing` flag. This builds your app for testing and also creates an `.xctestrun` file. 
+  
+  ### Configuring the Step 
+  
+  At a minimum, the Step needs valid values for three inputs:
+  
+  - **Project (or Workspace) path**: This is the path to the `.xcodeproj` or `.xcworkspace` file. In most cases, leave it on the default value.
+  - **Scheme name**: The name of your Xcode scheme. By default, the Step will use the scheme that was set when you added the app on Bitrise.
+  - **Device destination**: The device and platform type to build the tests for. For available values call, `man xcodebuild` and check the Destinations section. 
```

Comment:
I checked `man xcodebuild` and it doesn't have a destinations section anymore. A better place to check destinations is our system report: https://github.com/bitrise-io/bitrise.io/tree/master/system_reports
A destination is a combination of platform and device type.

Reply:
BTW, the step input has a reference to `man xcodebuild` too